Detectors/MUON/MCH/Simulation/include/MCHSimulation/GeometryTest.h

```diff
@@ -0,0 +1,52 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#ifndef O2_MCH_SIMULATION_GEOMETRYTEST_H
+#define O2_MCH_SIMULATION_GEOMETRYTEST_H
+
+#include <iostream>
```

Comment:
do you need this in the header? better to forward decl and do IO in source file

Reply:
hum, correction : I cannot forward declare and have the default value for the parameter : `std::ostream& out = std::cout`